backend/BusinessLogic/Services/VideoService/VideoService.cs

```diff
@@ -99,7 +99,7 @@ public async Task<Video> CompleteUpload(Guid userId, string fileName)
 
         public async Task DeleteVideo(Video video, Guid userId)
         {
-            string snapshotPath = Path.Combine(Path.Combine(_uploadPath, userId.ToString()), "Snapshots\\" + video.Id + ".png");
```

Comment:
General Comment: 
![image](https://user-images.githubusercontent.com/31959982/119029916-eb1f9e80-b9b1-11eb-8b4e-5d4dafd8a792.png)

It happens after restoring the last deleted video

Reply:
what the fuck